integration_tests/models/sql/test_unpivot_original_api.sql

```diff
@@ -0,0 +1,32 @@
+
+-- unpivot() was enhanced with 3 new parameters
+-- This test targets the original API.
+
+-- snowflake messes with these tests pretty badly since the
+-- output of the macro considers the casing of the source
+-- table columns. Using some hacks here to get this to work,
+-- but we should consider lowercasing the unpivot macro output
+-- at some point in the future for consistency
+
+{% if target.name == 'snowflake' %}
+    {% set exclude = ['CUSTOMER_ID', 'CREATED_AT'] %}
+{% else %}
+    {% set exclude = ['customer_id', 'created_at'] %}
+{% endif %}
+
+select
+    customer_id,
+    created_at,
+    case
+        when '{{ target.name }}' = 'snowflake' then lower(field_name)
+        else field_name
+    end as field_name,
```

Comment:
I think this is what caused the tests to fail. On Snowflake, this column is evaluated as `FIELD_NAME`, but the result of the `unpivot` macro is `field_name`. This is an error on Snowflake:
```
select field_name from (
  select 1 as "field_name"
)
```

Reply:
I think I may have fixed it but I can't get the CI to run, can you try @drewbanin?